tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners.Tests/MainTests.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Sdk.Tools.CodeOwnersParser;
+using NUnit.Framework;
+
+namespace Azure.Sdk.Tools.RetrieveCodeOwners.Tests
+{
+    public class MainTests
+    {
+        private const string codeOwnerFilePath = "CODEOWNERS";
+
+        [TestCase("sdk", false, new[] { "person1", "person2" })]
+        [TestCase("/sdk", false, new[] { "person1", "person2" })]
+        [TestCase("sdk/noPath", false, new[] { "person1", "person2" })]
+        [TestCase("/sdk/azconfig", false, new[] { "person3", "person4" })]
+        [TestCase("/sdk/azconfig/package", false, new[] { "person3", "person4" })]
+        [TestCase("/sdk/testUser/", true, new[] { "azure-sdk" })]
+        [TestCase("/sd", true, new string[0])]
+        public void testOnNormalOuput(string targetDirectory, bool includeUserAliasesOnly, List<string> expectedReturn)
+        {
+            
+            using (var consoleOutput = new ConsoleOutput())
+            {
+                Program.Main(codeOwnerFilePath, targetDirectory, includeUserAliasesOnly);
+                var output = consoleOutput.GetOuput();
+                testExpectResult(expectedReturn, output);
+                consoleOutput.Dispose();
+            }
+        }
+
+        [Test]
+        public void testOnError()
+        {
+            Assert.AreEqual(1, Program.Main("PathNotExist", "sdk"));
+        }
+
+        private void testExpectResult(List<string> expectReturn, string output)
```

Comment:
```suggestion
        private void TestExpectResult(List<string> expectReturn, string output)
```

Methods in .NET are supposed to start with capital letters by convention.